src/scss/src/foundation/_mixins.scss

```diff
@@ -1,16 +1,169 @@
+@use "sass:list";
 /*=============================================
 =                 #MIXINS                     =
 =============================================*/
 
-
-
 /**
  *
  * Helpful reusable functions-like code that will be used to implement
  * styles for components
  *
  */
 
+/*=============================================
+ =                @media rules                =
+ =============================================*/
+
+@mixin tablet {
+    @media (min-width: map-get($breakpoints, "tablet")) {
+        @content;
```

Comment:
Maybe you can use similar check for invalid argument like you did below?

```scss
@if not list.index($valid-properties, $property) {
    @error "#{$property} is not a valid property. Expected one of #{$valid-properties}.";
}
```

Reply:
These mixins are not accepting any parameters
you add it like
`@include tablet {
 color: pink;
}`